torch/_prims/context.py

```diff
@@ -95,3 +102,28 @@ def __torch_function__(
                 f"no _refs support for {torch.overrides.resolve_name(orig_func)}"
             )
         return orig_func(*args, **kwargs)
+
+
+def _is_node_supported_nvfuser(node):
+    return (
+        node.op == "call_function"
+        and getattr(node.target, "impl_nvfuser", None) is not None
+    )
+
+
+def _is_func_unsupported_nvfuser(torch_function_mode, func, args, kwargs):
+    with torch.overrides.enable_torch_function_mode(
+        torch_function_mode, replace=torch_function_mode.inner
+    ):
+        gm = get_isolated_graphmodule(func, args, kwargs)
+
+    call_function_nodes = filter(lambda n: n.op == "call_function", gm.graph.nodes)
+    any_unsupported = any(
+        not _is_node_supported_nvfuser(node) for node in call_function_nodes
+    )
+    return any_unsupported
+
+
+TorchRefsNvfuserCapabilityMode = functools.partial(
+    TorchRefsMode.push, should_fallback_fn=_is_func_unsupported_nvfuser
```

Comment:
per @samdow's recent PR you don't need `.push` here

Reply:
Thanks, I will change that once https://github.com/pytorch/pytorch/pull/78215 is actually merged.

Reply:
@IvanYashchuk It should work to change this now. That PR is just to clean up the API so that we insist on only having one way pushing a mode (and all code reflects that way)